core/src/test/java/io/questdb/griffin/AlterTableDropPartitionTest.java

```diff
@@ -42,7 +42,23 @@
 public class AlterTableDropPartitionTest extends AbstractGriffinTest {
```

Comment:
nit: would be great to add a few tests around bind variables in the dropped partition list.

Reply:
Are the tests in `PGJobContextTest` sufficient?

- testBindVariableDropLastPartitionListByNoneHigherPrecision
- testBindVariableDropLastPartitionListByMonthHigherPrecision
- testBindVariableDropLastPartitionListByWeekHigherPrecision
- testBindVariableDropLastPartitionListWithWeekPrecision

Reply:
I'd better add dedicated tests to proper bind variables support.

Reply:
I am not undestanding, wdym with "proper bind variables support" ?

Reply:
ok, I will then add an extra PR with just this alone, to double check.

Reply:
I consider this to be absolutely optional as the nit prefix suggests, so feel free to ignore this comment.